internal/node/controller/init_test.go

```diff
@@ -1,111 +0,0 @@
-package controller
-
-import (
```

Comment:
I don't see where these tests got moved to - are they still around?

Reply:
`testTransitions` is a copy of what's in transitions_test.go
TestInitTransitions just tests that there are 4 initial transitions ... which seems useless
the latter tests that the initial transitions include a redirect rule ... which also seems useless>

if we want to be testing something with that, we should probably have some exported defaults and test on those, rather than something that is just used in main.go directly